modules/ROOT/partials/events/tinymcespellchecker-events.adoc

```diff
@@ -9,4 +9,5 @@ The following events are provided by the xref:introduction-to-tiny-spellchecker.
 |SpellcheckStart |N/A |Fired when spellchecking is enabled.
 |SpellcheckEnd |N/A |Fired when spellchecking is disabled.
 |SpellcheckerLanguageChanged |`+{ language: string, prevLanguage: string }+` |Fired when the spellchecking language is changed.
+|SpellcheckerUpdated |`+{ spelling: SpellingUpdate }+` |Fired when the editor content is checked for misspellings and suggestions.
```

Comment:
```suggestion
|SpellcheckerUpdated |`+{ spelling: SpellingUpdate }+` |Fired when the spellchecker dialog is opened or when the `mceSpellcheckUpdate` command is executed. Not fired when the editor is in check spelling As-You-Type mode.
```
A somewhat more fulsome and specific description.

Reply:
I think this changes the original meaning, "when the editor content is checked for misspellings and suggestions" is more general whereas this suggestion, while correct, detracts from the purpose of the event. The other events in this table are provided in general terms, since the integrator will know when, but not how the events fire. Besides, this information is already conveyed under the "SpellcheckerUpdated Event" heading.

Reply:
I will update the content under that heading to provide more details to the integrator.